Source/Analytics/Web/src/components/Lightweight/LightweightAreaOverview.js

```diff
@@ -11,14 +12,19 @@ export default class LightweightAreaOverview extends Component {
 
     render() {
         return (
-            <div>
-                <CBSNavigation activeMenuItem="analytics/#" />
-                <div className="lightweight">
-                    <h2 className="header">Light Area Overview</h2>
-                    <p className="description">This is the light version of the country overview page. If you want the normal version click <Link to="/analytics">here</Link></p>
-                </div>
-                <HealthRiskSelector />
-            </div>
+            <Grid container>
+                <Grid item xs={12}>
+                    <CBSNavigation activeMenuItem="analytics/#" />
+                </Grid>
+                
+                <Grid container item xs={12} justify="center">
+                    <div className="lightweight">
+                        <h1>Light Area Overview</h1>
+                        <p>This is the light version of the country overview page. If you want the normal version click <Link to="/analytics">here</Link></p>
+                    </div>
+                    <HealthRiskSelector />
```

Comment:
Works well on small screens but the header and HealthRiskSelector are side by side on big screens, they should be centered and one below the other as for mobile.

Reply:
Well spotted:) Fixed it.